forms-flow-api/src/formsflow_api/services/draft.py

```diff
@@ -47,10 +46,20 @@ def create_new_draft(cls, application_payload, draft_payload, token, **kwargs):
                 f"Mapper does not exist with formId - {application_payload['form_id']}.",
                 HTTPStatus.BAD_REQUEST,
             )
+        if mapper.status == FormProcessMapperStatus.INACTIVE.value:
+            raise BusinessException(
```

Comment:
Can we reduce the same type of exception may be in a single if condition
if (mapper.status == FormProcessMapperStatus.INACTIVE.value) or (not token and not mapper.is_anonymous):

![image](https://user-images.githubusercontent.com/99173163/181511443-b13826df-1d13-41f7-b91b-4ac47eb4bc58.png)

Reply:
👍